modules/static/protocolbuffers/gofeatures/buf.md

```diff
@@ -1,5 +1,8 @@
 **This is a third-party repository managed by Buf.**
 
+This module has been deprecated in favor of `protocolbuffers/wellknowntypes`, which contains the
```

Comment:
This should specifically mention v29.2 -- if a user is using any older version then they can't use wellknowntypes instead.

Do you know if the Markdown rendering library we use in the BSR supports call-outs? If so, that would be nice, to make the deprecation visually obvious even if someone doesn't read all of the text.

In GitHub-flavored Markdown it would look something like so:
```md
>[!WARNING]
> This module has been **deprecated**. As of v29.2, use the `protocolbuffers/wellknowntypes` module instead.
```

With Docusaurus-style callouts, it might look like this instead:

```md
:::warning

This module has been **deprecated**. As of v29.2, use the `protocolbuffers/wellknowntypes` module instead.

:::
```

Reply:
> This should specifically mention v29.2 -- if a user is using any older version then they can't use wellknowntypes instead.

Sure, I'll say that.

> Do you know if the Markdown rendering library we use in the BSR supports call-outs? If so, that would be nice, to make the deprecation visually obvious even if someone doesn't read all of the text.

cc @paul-sachs which one should we use for BSR Readmes?

Reply:
I don't think so. We're just using GitHub flavoured markdown and I'm not aware of call-out support within that variant.

Reply:
<img width="702" alt="SCR-20241219-mscu" src="https://github.com/user-attachments/assets/2f054f31-0a97-4ba5-973a-11eb179818dc" />

Tried locally, as Paul says I don't think there's a way to do call-outs 😞 so I think just a quote or H2/H3 is good enough?